addons/advanced_fatigue/functions/fnc_mainLoop.sqf

```diff
@@ -53,7 +53,9 @@ GVAR(ae1Reserve) = GVAR(ae1Reserve) - _ae1Power / WATTSPERATP;
 GVAR(ae2Reserve) = GVAR(ae2Reserve) - _ae2Power / WATTSPERATP;
 GVAR(anReserve)  = GVAR(anReserve)  -  _anPower / WATTSPERATP;
 // Increase anearobic fatigue
-GVAR(anFatigue)  = GVAR(anFatigue) + _anPower * (0.057 / GVAR(peakPower)) * 1.1;
+// 1264.64 = 0.057 / peakPower with default VO2 max
+// See #6163
+GVAR(anFatigue)  = GVAR(anFatigue) + _anPower * 1264.64 * 1.1;
```

Comment:
`1264.64` seems to be very high. Where did you get peakPower `0.00004507211` from?